src/main/resources/standardTestCases/monitoring_log/monitoring_log.options

```diff
@@ -0,0 +1,3 @@
+{
+    "monitoring_script": "gs://cloud-cromwell-dev/some/faux_script.sh"
```

Comment:
It's faux because the script is minimalistic and it only ever prints out [cpu/mem/disk](https://github.com/broadinstitute/centaur/blob/rm_monitoring/src/main/resources/standardTestCases/monitoring_log.test#L15-L17) availability once. The naming is to indicate that it's a fake script to test that *something* is run in the background and the output of that script is not null. In the case of the `monitoring_script` workflow option, Cromwell sends the output of that script to a file named "monitoring.log". Ideally, I'd want to do a `read_string` function on the monitoring.log file, kind of like how we can do `read_string(stdout())`. But since I can't do that, I do know what the monitoring output file is called, so I'm just reading that instead directly by filename.